stdlib/LinearAlgebra/test/symmetric.jl

```diff
@@ -517,4 +517,13 @@ end
     @test Hermitian(A, :U)[1,1] == Hermitian(A, :L)[1,1] == real(A[1,1])
 end
 
+@testset "issue #29392" begin
+    S = Symmetric(rand(Float64, 2, 2))
+    H = Hermitian(rand(ComplexF64, 2, 2))
+    @test (2.0 * S)::Matrix{Float64} == (S * 2.0)::Matrix{Float64}
+    @test (2.0 * H)::Matrix{ComplexF64} == (H * 2.0)::Matrix{ComplexF64}
+    @test (S / 2.0)::Matrix{Float64} == Matrix(S) / 2.0
+    @test (H / 2.0)::Matrix{ComplexF64} == Matrix(H) / 2.0
+end
```

Comment:
I imagine I'm missing something obvious: Why not preserve the structural information? :)

Reply:
Are we trying to make `Symmetric`/`Hermitian` full fledged abstract arrays? I feel like that can never be complete, there will always be something that does not return a `Symmetric`. Personally I just consider them dispatch wrappers, and same with `Adjoint`/`Transpose`, but I guess I am alone in this thinking.

Anyway, I guess this change was not necessary and maybe it is better to make `::Number * ::SymOrHerm` preserve the structural property?

Reply:
> I feel like that can never be complete

There are a finite number of operations defined in Base, so this can't be true. We're still in the phase where we're encountering things we missed all the time, but it's getting less and less over time and will eventually be none. If it's still a common problem in a couple of minor versions (say 1.3) then, we can consider if it's just not working, but it seems premature to declare failure at this point.